src/utilities/_fonts.scss

```diff
@@ -0,0 +1,13 @@
+@font-face {
+    font-family: 'Heavitas';
+    src: url('utilities/fonts/heavitas-webfont.woff2') format('woff2'),
+         url('utilities/fonts/heavitas-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+
```

Comment:
whitespace?

Reply:
Whitespace              is               nice.